tsfresh/feature_extraction/extraction.py

```diff
@@ -151,8 +153,11 @@ def _extract_features_parallel_per_kind(kind_to_df_map, settings, column_id, col
                                                            settings=settings)
     pool = Pool(settings.n_processes)
 
-    extracted_features = pool.map(partial_extract_features_for_one_time_series, kind_to_df_map.items(),
-                                  chunksize=settings.chunksize)
+    chunksize = _calculate_best_chunksize(kind_to_df_map, settings)
+
+    total_number_of_expected_results = len(kind_to_df_map)
+    extracted_features = tqdm(pool.imap(partial_extract_features_for_one_time_series, kind_to_df_map.items(),
```

Comment:
We can just add a flag to the almighty settings object to steer if tqdm is used or not, right?